functions-and-operators/json-functions.md

```diff
@@ -93,4 +109,3 @@ The following JSON functions are unsupported in TiDB. You can track the progress
 [json_append]: https://dev.mysql.com/doc/refman/5.7/en/json-modification-functions.html#function_json-append
 [json_array_append]: https://dev.mysql.com/doc/refman/5.7/en/json-modification-functions.html#function_json-array-append
 [json_array_insert]: https://dev.mysql.com/doc/refman/5.7/en/json-modification-functions.html#function_json-array-insert
-[json_search]: https://dev.mysql.com/doc/refman/5.7/en/json-search-functions.html#function_json-search
```

Comment:
Is this removed because we don't support it?

Reply:
No, it's mentioned twice.